app/controllers/admin/rdvs_controller.rb

```diff
@@ -81,7 +81,8 @@ def create
         notice: "Le rendez-vous a été créé."
       )
     else
-      @rdv_wizard = AgentRdvWizard::Step3.new(current_agent, current_organisation, @rdv.attributes)
+      @rdv_wizard = AgentRdvWizard::Step3.new(current_agent, current_organisation, users: @rdv.users, agent_ids: @rdv.agent_ids, **@rdv.attributes)
+      @rdv_wizard.valid? # trigger validation
```

Comment:
Je crois que ce commentaire n'est pas nécessaire :smile:

Reply:
je me disais que ca peut surprendre cette ligne parce que généralement on appelle une méthode en `?` pour récupérer la valeur, tandis qu'ici c'est pour délencher le side effect de peupler les `errors` et `warnings`. 
Ce hack est en fait du au fait qu'on n'appelle pas `save` sur l'objet formulaire `rdv_wizard` mais sur le rdv, il y aurait un petit micmac à démêler ici. 

je vais peut etre reformuler le commentaire en disant `necessary to explicitly trigger validations here so that errors and warnings appear in the view`, ca te va ?

Reply:
moi ça me va sans commentaire :) Si tu veux l'enrichir, ça ne me dérange pas. 

J'ai une autre suggestion : avoir une méthode sur l'instance `@rdv_wizard` qui ferait genre : `@rdv_wizard.trigger_validations` et qui serais un alias pour `@rdv_wizard.valid?` :smile: 

:warning: _explicit code inside_ :smile:

Reply:
arf je pense que le "bon fix" serait de remplacer @rdv.save par @rdv_wizard.save plutot, et que ca delegue du form object vers le modele AR . je vais laisser le commentaire en plus explicite pour l'instant !